perfkitbenchmarker/providers/gcp/gcp_bigtable.py

```diff
@@ -26,42 +26,47 @@
 FLAGS = flags.FLAGS
 
 
-class GcpBigtableCluster(resource.BaseResource):
-  """Object representing a GCP Bigtable Cluster.
+class GcpBigtableInstance(resource.BaseResource):
+  """Object representing a GCP Bigtable Instance.
 
   Attributes:
-    name: Cluster name.
-    num_nodes: Number of nodes in the cluster.
-    project: Enclosing project for the cluster.
-    zone: zone of the cluster.
+    name: Instance and cluster name.
+    num_nodes: Number of nodes in the instance's cluster.
+    project: Enclosing project for the instance.
+    zone: zone of the instance's cluster.
   """
 
   def __init__(self, name, num_nodes, project, zone):
-    super(GcpBigtableCluster, self).__init__()
+    super(GcpBigtableInstance, self).__init__()
     self.num_nodes = num_nodes
     self.name = name
     self.zone = zone
     self.project = project
 
   def _Create(self):
-    """Creates the cluster."""
-    cmd = util.GcloudCommand(self, 'alpha', 'bigtable', 'clusters', 'create',
+    """Creates the instance."""
+    cmd = util.GcloudCommand(self, 'beta', 'bigtable', 'instances', 'create',
                              self.name)
     cmd.flags['description'] = 'PkbCreatedCluster'
-    cmd.flags['nodes'] = str(self.num_nodes)
-    cmd.flags['zone'] = self.zone
+    cmd.flags['cluster'] = self.name
+    cmd.flags['cluster-num-nodes'] = str(self.num_nodes)
+    cmd.flags['cluster-zone'] = self.zone
     cmd.flags['project'] = self.project
+    # The zone flag makes this command fail.
```

Comment:
Yes, I believe it's added here:
https://github.com/GoogleCloudPlatform/PerfKitBenchmarker/blob/master/perfkitbenchmarker/providers/gcp/util.py#L114